java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -493,4 +493,22 @@ CompletableFuture<Map<String, Map<String, String[][]>>> xreadgroup(
             String group,
             String consumer,
             StreamReadGroupOptions options);
+
+    /**
+     * Returns the number of messages that were successfully acknowledged by the consumer group member of a stream.
+     * This command should be called on a pending message so that such message does not get processed again.
+     *
+     * @param key The key of the stream.
+     * @param group The consumer group name.
+     * @param ids Stream entry ID to acknowledge and purge messages.
+     * @return The number of messages that were successfully acknowledged.
+     * @example
+     *     <pre>{@code
+     * String entryId = client.xadd("mystream", Map.of("myfield", "mydata")).get();
+     * // read and process messages from streamId
+     * assert 1L == client.xack("mystream", "mygroup", new String[] {entryId}).get();
```

Comment:
I think this will fail. From my understanding, we need to run `XREADGROUP` after `XADD` in order for the entry to be added to the PEL.